contrib/buildrpm.sh

```diff
@@ -57,6 +60,10 @@ if [ $opt_strict_ibverb_dep -eq 1 ]; then
     rpmmacros="${rpmmacros} --define='extra_deps libibverbs == ${libibverbs_ver}'"
 fi
 
+if [ $opt_dbgrpm -eq 1 ]; then
+    rpmmacros="--define 'devel 1' ${rpmmacros}"
```

Comment:
can we rename "devel" -> "debug"?
for consistency

Reply:
Done